op-chain-ops/cmd/celo-migrate/ancients.go

```diff
@@ -152,6 +155,12 @@ func transformBlocks(ctx context.Context, in <-chan RLPBlockRange, out chan<- RL
 			for i := range blockRange.hashes {
 				blockNumber := blockRange.start + uint64(i)
 
+				if blockNumber != 0 && blockNumber != prevBlockNumber+1 {
```

Comment:
This check for zero is redundant now
```suggestion
				if blockNumber != prevBlockNumber+1 {
```